### PR TITLE
Updates for upgrades component eval

### PR DESCRIPTION
**Pull request by vsedmik**
### Problem Statement
During component eval some changes were proposed by PX/SD/QE for upgrade tests (scenario and all-tiers).

### Solution
This PR just implements a few of them, mostly related to @SatelliteQE/phoenix 
For reference check the _Component feedback sheet - Team Platform_ g-sheet

Split into a bunch of tiny commits, one per each g-sheet line 

### PRT test Cases example
trigger: test-robottelo
pytest: tests/foreman/cli/test_activationkey.py -k test_positive_update_aks_to_chost

### CODE REVIEW

**Comment by vsedmik** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_activationkey.py -k test_positive_update_aks_to_chost

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7396
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/cli/test_activationkey.py -k test_positive_update_aks_to_chost --external-logging
Test Result : ========= 3 failed, 145 deselected, 176 warnings in 711.54s (0:11:51) ==========
```

**Comment by vsedmik** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_activationkey.py -k test_positive_update_aks_to_chost

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7398
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_activationkey.py -k test_positive_update_aks_to_chost --external-logging
Test Result : ========= 3 passed, 145 deselected, 178 warnings in 760.60s (0:12:40) ==========
```

File: tests/foreman/cli/test_activationkey.py [hunk 1]
```diff
@@ -874,14 +874,13 @@ def test_positive_delete_subscription(function_entitlement_manifest_org, module_
 @pytest.mark.skip_if_not_set('clients')
 @pytest.mark.tier3
 @pytest.mark.upgrade
-def test_positive_update_aks_to_chost(module_org, module_location, rhel7_contenthost, target_sat):
-    """Check if multiple Activation keys can be attached to a
-    Content host
+@pytest.mark.rhel_ver_match('[^6]')
+def test_positive_update_aks_to_chost(module_org, module_location, rhel_contenthost, target_sat):
```
Comment (jameerpathan111) [inline tests/foreman/cli/test_activationkey.py:878]:
```suggestion
def test_positive_update_aks_to_chost(module_org, module_location, rhel_contenthost, module_target_sat):
```
To be consistent, I'd suggest using `module_target_sat` in the test.

File: tests/foreman/cli/test_contentaccess.py [hunk 2]
```diff
@@ -205,13 +203,12 @@ def test_negative_rct_not_shows_golden_ticket_enabled(
 
 @pytest.mark.tier2
 @pytest.mark.upgrade
-def test_positive_rct_shows_golden_ticket_enabled(module_sca_manifest, target_sat):
-    """Assert unrestricted manifest has Golden Ticket enabled .
+def test_positive_rct_shows_sca_enabled(module_sca_manifest, target_sat):
```
Comment (jameerpathan111) [inline tests/foreman/cli/test_contentaccess.py:206]:
```suggestion
def test_positive_rct_shows_sca_enabled(module_sca_manifest, module_target_sat):
```

**Review by jameerpathan111 — CHANGES_REQUESTED** [review]

**Comment by vsedmik** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_activationkey.py -k test_positive_update_aks_to_chost

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7441
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_activationkey.py -k test_positive_update_aks_to_chost --external-logging
Test Result : ========= 3 passed, 145 deselected, 177 warnings in 797.63s (0:13:17) ==========
```

**Review by jameerpathan111 — APPROVED** [review]

**Review by pondrejk — APPROVED** [review]

**Review by sambible — APPROVED** [review]

**Pull request merged**